Dangerfile

```diff
@@ -12,24 +12,24 @@ require 'json'
 
 containing_dir = ENV["CIRCLE_ARTIFACTS"] || "." # for local testing
 rspec_files = Dir[File.join(containing_dir, "rspec_logs_*.json")]
-fail("Could not find test artifacts") if rspec_files.count == 0
+if rspec_files.count > 0
```

Comment:
The contents of the `each` iteration should be skipped if the count is zero either way. No check needed here! :+1: